Multisig/UI/Safe Management/SafeAddressForm/SafeAddressForm.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  SafeAddressForm.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 24.04.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct SafeAddressForm: View {
```

Comment:
We will need to discuss view names. I would prefer to have them aligned with Sketch to reduce the cognitive load. For me, a form is usually a part of a view, but not the view itself.

Reply:
For me, the Form is just a form, similar to a website form. It is a view. 🤷 

For the cognitive load, this is a valid point. I'll see what we can do.